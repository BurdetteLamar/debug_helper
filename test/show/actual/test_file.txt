---
File (message='test_file'):
  File.executable?("C:/Users/Burde/Documents/GitHub/debug_helper/test/debug_helper_test.rb"): false
  File.exist?("C:/Users/Burde/Documents/GitHub/debug_helper/test/debug_helper_test.rb"): true
  File.ftype("C:/Users/Burde/Documents/GitHub/debug_helper/test/debug_helper_test.rb"): file
  File.pipe?("C:/Users/Burde/Documents/GitHub/debug_helper/test/debug_helper_test.rb"): false
  File.readable?("C:/Users/Burde/Documents/GitHub/debug_helper/test/debug_helper_test.rb"): true
  File.setgid?("C:/Users/Burde/Documents/GitHub/debug_helper/test/debug_helper_test.rb"): false
  File.setuid?("C:/Users/Burde/Documents/GitHub/debug_helper/test/debug_helper_test.rb"): false
  File.socket?("C:/Users/Burde/Documents/GitHub/debug_helper/test/debug_helper_test.rb"): false
  File.symlink?("C:/Users/Burde/Documents/GitHub/debug_helper/test/debug_helper_test.rb"): false
  File.writable?("C:/Users/Burde/Documents/GitHub/debug_helper/test/debug_helper_test.rb"): true
