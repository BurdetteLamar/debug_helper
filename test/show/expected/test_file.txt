---
File (message='test_file'):
  File.executable?("git_dir/test/debug_helper_test.rb"): false
  File.exist?("git_dir/test/debug_helper_test.rb"): true
  File.ftype("git_dir/test/debug_helper_test.rb"): file
  File.pipe?("git_dir/test/debug_helper_test.rb"): false
  File.readable?("git_dir/test/debug_helper_test.rb"): true
  File.setgid?("git_dir/test/debug_helper_test.rb"): false
  File.setuid?("git_dir/test/debug_helper_test.rb"): false
  File.socket?("git_dir/test/debug_helper_test.rb"): false
  File.symlink?("git_dir/test/debug_helper_test.rb"): false
  File.writable?("git_dir/test/debug_helper_test.rb"): true
